Project/Mac/Info.plist

```diff
@@ -19,11 +19,11 @@
 	<key>CFBundleVersion</key>
 	<string>25.03</string>
 	<key>DTSDKName</key>
-	<string>macosx10.10</string>
+	<string>macosx10.12</string>
```

Comment:
Too complicated to keep macosx10.10 compatibility?

Reply:
Already dropped in previous release (cannot use Qt5.12 with recent Xcode versions)

Reply:
@g-maxime please add the latest 10.10 compatible version to https://mediaarea.net/BWFMetaEdit/Download/Mac_OS